cmds/clients/contestcli-http/main.go

```diff
@@ -77,26 +81,56 @@ func run(verb string) error {
 			return fmt.Errorf("failed to parse job descriptor: %v", err)
 		}
 		params.Add("jobDesc", string(jobDesc))
+		resp, err := request(verb, params)
+		if err != nil {
+			return err
+		}
+		fmt.Println(resp)
+		// if wait flag set, keep polling for status till job is completed
+		if *flagWait {
+			fmt.Fprintf(os.Stderr, "\nWaiting for job to complete...")
+			var parsedResp map[string]interface{}
```

Comment:
this should also be a typed structure as I suggested below, after changing the signature of `request()`, so there's no need for the typecasting dance a few lines below

Reply:
Done